modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/resources/content/Language.properties

```diff
@@ -1,10 +1,31 @@
+anonymize-data=Anonymize Data
+application-data-review=Application Data Review
 auto-anonymize=Auto Anonymize
 auto-anonymize-all=Auto Anonymize All
+deactivate-user=Deactivate User
 delete-all=Delete All
+delete-personal-site=Delete Personal Site
+delete-user=Delete User
 edit-url=Edit URL
 entity-id=Entity ID
 erase-personal-data=Erase Personal Data
 export-all=Export All
+forget-user-data=Forget User Data
 no-data-requires-anonymization=No data requires anonymization.
 no-entities-remain-of-this-type=No entities remain of this type.
-nonanonymizable-fields=Nonanonymizable Fields
\ No newline at end of file
+nonanonymizable-fields=Nonanonymizable Fields
+open-dashboard-pages=Open Dashboard Pages
+open-profile-pages=Open Profile Pages
+personal-data-erasure=Personal Data Erasure
+personal-site-successfully-forgotten=Personal site successfully forgotten
+step-five-description=Delete the user to complete the data erasure process.
```

Comment:
I just talked with Brian, and he's fine with these `step-*` keys being semantic (like they are now).  However, since the keys themselves are a little generic (they could show up in the core language files at some point), let's prefix them all with `summary-page-step-*`.